### ISSUE TITLE
SelectOneRadio: keyboard navigation does not work with Safari

### ISSUE THREAD

**Issue by jepsar** [primefaces/primefaces#8408]
See https://www.primefaces.org/showcase/ui/input/oneRadio.xhtml and try to tab through the radio groups / buttons.

This should be addressed before #5503

**Comment by jepsar** [comment]
@melloware I don't know if you remember this one, but I was fooled again by this Safari feature! By default it lets you focus certain elements when using (Shift+)Tab. If you want to focus each focusable element, you'll need to use Option+(Shift+)Tab.

**Comment by melloware** [comment]
Ahh yes I remember we got burned by this before!

**Comment by jepsar** [comment]
FWIW #7416
